README.md

```diff
@@ -142,41 +192,38 @@ Sometimes you may want to stop using Pi-hole or turn it back on.  You can trigge
+##### Query Ad Lists
+You can find out what blocklist a specific domain was found on. This is useful for troubleshooting websites that may not work properly due to a blocked domain.
 
-#### `tail`ing Log Files
-You can [watch the log files](https://discourse.pi-hole.net/t/how-do-i-watch-and-interpret-the-pihole-log-file/276) in real time to help debug any issues, or just see what's happening with your Pi-hole.
+##### `tail`ing Log Files
+You can [watch the log files](https://discourse.pi-hole.net/t/how-do-i-watch-and-interpret-the-pihole-log-file/276) in real time to help debug any issues, or just see what's happening on your network.
 
-#### Pi-hole Debugger
-If you are having trouble with your Pi-hole, this is the place to go.  You can run the debugger and it will attempt to diagnose any issues and then link to an FAQ with instructions on rectifying the problem.
+##### Pi-hole Debugger
+If you are having trouble with your Pi-hole, this is the place to go. You can run the debugger and it will attempt to diagnose any issues, and then link to an FAQ with instructions on rectifying the problem.
 
 <p align="center">
 <img src="https://assets.pi-hole.net/static/debug-gui.png">
 </p>
 
-If run [via the command line](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug), you will see red/yellow/green text, which makes it easy to identify any problems.
+If run [via the command line](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug), you will see coloured text, which makes it easy to identify any problems.
 
 <p align="center">
 <a href=https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debugs><img src="https://assets.pi-hole.net/static/debug-cli.png"></a>
 </p>
 
-
-After the debugger has finished, you have the option to upload it to our secure server for 48 hours.  All you need to do then is provide one of our developers the unique token generated by the debugger (this is usually done via [our forums](https://discourse.pi-hole.net/c/bugs-problems-issues)).
+After the debugger has finished, you have the option to upload it to our secure server for 48 hours. All you need to do is provide [one of our developers](https://github.com/orgs/pi-hole/teams/debug/members) the unique token generated by the debugger via [one of the various ways of getting in touch with us](#getting-in-touch-with-us).
 
 <p align="center">
 <a href=https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debugs><img src="https://assets.pi-hole.net/static/debug-token.png"></a>
 </p>
 
-However, most of the time, you will be able to solve any issues without any intervention from us.  But if you can't, we're always around to help out.
+You should be able to resolve most issues using the provided FAQ links, but we're always happy to help out if you'd like assistance!
 
-### Settings
-
-The settings page lets you control and configure your Pi-hole.  You can do things like:
+### Web Admin Settings
```

Comment:
Web Interface Settings

Reply:
Looking back over prior discussion on this exact topic, you're absolutely correct.